src/GeolocatorService/MockedLocationService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Devices.Geolocation;
+
+namespace GeolocatorService
+{
+	/// <inheritdoc cref="IGeolocatorService"/>
+	public class MockedLocationService : IGeolocatorService
+	{
+		private bool _isPermissionRequested;
+		private bool _isPermissionAllowedWhenRequested;
+		private Geocoordinate _geoCoordinate;
+
+		public MockedLocationService(bool isPermissionAllowedWhenRequested, Geocoordinate geoCoordinate)
+		{
+			_isPermissionAllowedWhenRequested = isPermissionAllowedWhenRequested;
+			_geoCoordinate = geoCoordinate;
+		}
+
+		public event LocationChangedEventHandler LocationChanged;
```

Comment:
It would be nice to add public setters on this class to be able to test the events.